packages/unigraph-dev-common/src/types/unigraph.ts

```diff
@@ -309,7 +309,7 @@ export interface Unigraph<TT = WebSocket | false> {
     /** Gets a list of schemas. */
     getSchemas(schemas?: string[] | undefined, resolve?: boolean): Promise<Record<string, SchemaDgraph>>;
     getObject?(
-        uidOrName: string,
+        uid: string,
```

Comment:
This is intentional, in the `localUnigraphApi` it will try to resolve named entity (such as `$/entity/inbox`) to its UID